Modules/Exercise05/Tests/OrderServiceTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Modules\Exercise05\Tests;
+
+use Modules\Exercise05\Services\OrderService;
+use Tests\TestCase;
+
+class OrderServiceTest extends TestCase
+{
+    /**
+     * @dataProvider detail_order_input
+     * @param $detailOrder
+     * @param $expectedResult
+     */
+    public function test_handle_discount($detailOrder, $expectedResult)
+    {
+        $service = new OrderService();
+        $result = $service->handleDiscount($detailOrder);
+        $this->assertEquals($expectedResult, $result);
+    }
+
+    public function detail_order_input()
```

Comment:
Trong provider này nên thêm có 1 case nữa check độ chính xác tới 2 chữ số thập phân vì cái round nó có thêm tham số thứ 2 là `2`
```
$infoBill['price'] = round(($detailOrder['price'] * 80) / 100, 2);
```
Cái này nice to have :smile: